tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+"""
+# -*- coding: utf-8 -*-
+This is used to configure the pytest environment.
+"""
+
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--run-integration",
+        action="store_true",
+        default=False,
+        help="run integration tests",
+    )
+
+
+def pytest_runtest_setup(item):
```

Comment:
I added this such that the integration tests can be run with a flag.

while spinning up the services locally.